fairmq/shmem/Manager.cxx

```diff
@@ -11,9 +11,25 @@
 // Needed to compile-firewall the <boost/process/async.hpp> header because it
 // interferes with the <asio/buffer.hpp> header. So, let's factor
 // the whole dependency to Boost.Process out of the header.
+#include <boost/version.hpp>
 #include <boost/process.hpp>
 #include <fairlogger/Logger.h>
 
+// Boost 1.88+ compatibility: boost::process v2 is now default, v1 APIs moved to v1 namespace
+// See: https://github.com/boostorg/process/issues/480
+#if BOOST_VERSION >= 108800
+namespace boost {
+// Provide compatibility aliases for boost::process v1 API
+namespace this_process = process::v1::this_process;
+namespace process_v1 = process::v1;
+}
+#else
+namespace boost {
+// For older Boost versions, process_v1 points to the default process namespace
+namespace process_v1 = process;
+}
+#endif
```

Comment:
Do not define new names in the upstream boost namespace. Rather define alias names in `fair::mq` and point them conditionally to the correct upstream names.